src/scripts/map.ts

```diff
@@ -29,21 +40,26 @@ export function mapListeners(
   sevenDayDiv: HTMLElement,
   openMap: HTMLElement
 ): void {
-  openMap.addEventListener("click", () =>
-    mapDisplayToggle(mapDiv, sevenDayDiv)
-  );
+  // openMap.addEventListener("click", () =>
+  //   mapDisplayToggle(mapDiv, sevenDayDiv)
+  // );
+
+  const mapButton$ = fromEvent(openMap, "click");
+  mapButton$.subscribe(() => {
+    mapDisplayToggle(mapDiv, sevenDayDiv);
+  });
 
   map.on("click", onMapClick);
 
-  function onMapClick(e) {
+  function onMapClick(e: any) {
```

Comment:
Not any

Reply:
`function onMapClick(e: { latlng: { lat: number; lng: number } }) `